src/DataFixtures/SpecialSponsorshipFixtures.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DataFixtures;
+
+use App\Entity\SpecialSponsorship;
+use App\Repository\SpecialSponsorship\SpecialSponsorshipManagerInterface;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use Faker\Factory as Faker;
+
+final class SpecialSponsorshipFixtures extends Fixture
+{
+    public const NB_SPECIAL_SPONSORSHIP = 10;
+
+    private $specialSponsorshipManager;
+
+    public function __construct(SpecialSponsorshipManagerInterface $specialSponsorshipManager)
+    {
+        $this->specialSponsorshipManager = $specialSponsorshipManager;
+    }
+
+    /**
+     * @param ObjectManager $manager
+     * @throws \Exception
+     */
+    public function load(ObjectManager $manager)
+    {
+        $faker = Faker::create();
+        for ($ss = 0; $ss < self::NB_SPECIAL_SPONSORSHIP; ++$ss) {
```

Comment:
`$ss` is not a great name here

Reply:
What kind of variable do you expect here ? $nbSpecialSponsorship ?

Reply:
`$iterationNbr` or something similar, while a bit technical, sounds fine enough.
`$specialSponsorshipNbr` is quite nice as well (just reverse the `Nbr` part from your suggestion)